executor/runtime/docker/docker.go

```diff
@@ -2486,7 +2495,9 @@ func (r *DockerRuntime) setupPostStartNetworkingAndIsolate(parentCtx context.Con
 	r.registerRuntimeCleanup(func() error {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
-		if err := stopSystemServices(ctx, c); err != nil {
+		err := stopSystemServices(ctx, r.dbusConn, c)
+		r.dbusConn.Close()
```

Comment:
Is the dbus connection held open for the lifetime of the container when it wasn't before?

Reply:
Yes, that's what I was trying to say in the commit message, but didn't say it as clearly. That's why I don't think we should merge this -- there's really no need to have this connection open.